Doc/tutorial/errors.rst

```diff
@@ -159,16 +201,13 @@ then re-raise the exception (allowing a caller to handle the exception as well):
        s = f.readline()
        i = int(s.strip())
    except OSError as err:
-       print("OS error: {0}".format(err))
+       print(f"OS error: {err}")
```

Comment:
```suggestion
       print("OS error:", err)
```

I often see that people love f-strings and forget that `print` is amazing!  Accepts multiple params, converts them to strings, adds a space or something else between them, all that.